flink-rpc/flink-rpc-akka/pom.xml

```diff
@@ -136,21 +123,11 @@ under the License.
 
 	<dependencyManagement>
 		<dependencies>
-			<dependency>
-				<groupId>org.scala-lang</groupId>
-				<artifactId>scala-compiler</artifactId>
-				<version>${scala.version}</version>
-			</dependency>
 			<dependency>
 				<groupId>org.scala-lang</groupId>
 				<artifactId>scala-library</artifactId>
```

Comment:
Why do we need `scala-library` still as a dependency here? :thinking: There are no sub-modules that could use it

Reply:
We need to overwrite the version here else we'd pull in what the parent pom declares (2.12.7)